lib/ansible/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -181,6 +181,19 @@
         required: false
         default: null
         description: The CA bundle string with custom certificates. defaults to None.
+
+  auth_type:
+    required: false
+    description: Type of auth
```

Comment:
you need a more full description here, people that do not know about OpenStack Director can get really confused here !

Reply:
Good point, have updated.